src/collection-preferences/visible-content.scss

```diff
@@ -38,7 +38,7 @@ $border: awsui.$border-divider-list-width solid awsui.$color-border-divider-defa
 }
 
 .visible-content-option-label {
-  white-space: nowrap;
+  white-space: normal;
```

Comment:
Since this is the initial value of the CSS (see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space#formal_definition)), I think we can omit this line now